### ISSUE TITLE
Add command line options to kernel installers

### ISSUE THREAD

**Issue by jeffyjefflabs** [nbgallery/jupyter-docker#5]
Some subset of...
- --reinstall: run the install block again, in case of partial failure
- --upgrade: update the appropriate apks, etc
- --install-only: don't call the real executable at the end of the installer

**Comment by kylekyle** [comment]
Yes, because adding the apk isn't enough - there's a config flag that must be set for Jupyter to recognize the kernel.
